src/main/webapp/app/exercises/shared/rating/rating.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, Input } from '@angular/core';
+import { RatingService } from 'app/exercises/shared/rating/rating.service';
+import { StarRatingComponent } from 'ng-starrating';
+import { Result } from 'app/entities/result.model';
+import { Rating } from 'app/entities/rating.model';
+
+@Component({
+    selector: 'jhi-rating',
+    providers: [RatingService],
+    templateUrl: './rating.component.html',
+    styleUrls: ['./rating.component.scss'],
+})
+export class RatingComponent {
+    // public ratingValue = 2;
+    public rating: Rating;
+    private _result: Result;
```

Comment:
I don't think we use this syntax of prefixing private attributes with an underscore at the moment.